src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -1003,6 +1003,12 @@ export default {
       }
     },
 
+    filingAuthor (header: HeaderIF): string {
+      if (header.isStaff) return 'Registry Staff'
+      if (header.certifiedBy) return header.certifiedBy
+      return 'Unknown'
+    },
```

Comment:
I know y'all loved the ternary (with a || thrown in) but I think THIS code is easier to understand 😁